pallets/time-release/src/types.rs

```diff
@@ -27,7 +27,10 @@ pub type ScheduleName = [u8; 32];
 	MaxEncodedLen,
 	TypeInfo,
 )]
-pub struct ReleaseSchedule<BlockNumber, Balance: MaxEncodedLen + HasCompact> {
+pub struct ReleaseSchedule<BlockNumber, Balance>
+where
+	Balance: MaxEncodedLen + HasCompact,
```

Comment:
Anytime you have more than one, it is now suggested to do a `where` to reduce the complexity of the signature. I agree.